src/components/no-run-status/no-run-status.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ExportModal from '../export-modal';
+import MetaData from '../metadata';
+import MetadataModal from '../metadata-modal';
+import Sidebar from '../sidebar';
+import SadSmiley from '../icons/sad-smiley';
+
+import './no-run-status.scss';
+
+export const NoRunStatus = ({
+  displaySidebar,
+  displayMetadataPanel,
+  displayExportBtn,
+}) => {
+  return (
+    <div className="kedro-pipeline">
```

Comment:
I think rather than writing the whole component from scratch, we just re-use the existing component here. For example `PipelineWarningContent`

Reply:
DONE